build_projects/dotnet-host-build/build.sh

```diff
@@ -93,8 +93,8 @@ done < "$REPOROOT/branchinfo.txt"
 [ -z "$DOTNET_INSTALL_DIR" ] && export DOTNET_INSTALL_DIR=$REPOROOT/.dotnet_stage0/$(uname)
 [ -d "$DOTNET_INSTALL_DIR" ] || mkdir -p $DOTNET_INSTALL_DIR
 
-
-$REPOROOT/scripts/obtain/dotnet-install.sh --channel preview
+DOTNET_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.sh"
+curl -sSL "$DOTNET_INSTALL_SCRIPT_URL" | bash /dev/stdin --channel preview --verbose
```

Comment:
When you make this change, this should work. Core-Setup had an older version of the install script because of this bug, the new version doesn't work with the older channels.